src/database/seeders/20191112083122-demo-users.js

```diff
@@ -1,10 +1,16 @@
-const hash = require('bcrypt');
```

Comment:
I think you might have to convert these back to ES6 so that we can run seeders on Heroku. Kindly liaise with our DevOps guy for this.